pkg/controller/migration/namespace_migration.go

```diff
@@ -615,107 +616,6 @@ func (m *CoreNamespaceMigration) getNumPodsDesiredAndReady(ctx context.Context,
 		nil
 }
 
-// addNodeLabels adds the specified labels to the named node. Perform
```

Comment:
I think we can put these back where they were since we're not longer using them. Let's minimize unnecessary changes in this set.

Reply:
Fixed in 26191326aa3d8e0393d4f89f88c606ffee8f285a